src/pages/[lang]/places/[uuid]/index.astro

```diff
@@ -15,7 +15,35 @@ const { lang, uuid } = Astro.params;
 const { place } = await PlacesService.getFull(uuid);
 const excludes = config.search?.result_filtering?.places?.exclude || [];
 
-export const getStaticPaths = async () => await getDetailPagePaths(config, ModelNames.places);
+let coverUrl = null;
+
+const firstManifest = place.relatedRecords.manifests?.items;
+
+if (firstManifest && firstManifest.length > 0) {
```

Comment:
The public Core Data API has a `/model/:uuid/media_contents` route that would probably be more straight forward to use than parsing the manifest. We'd also need to update the content loaders, services, and API.